pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -644,6 +644,14 @@ type ComponentSpec struct {
 	// - SLOW_LOG_FILE
 	// +optional
 	Env []corev1.EnvVar `json:"env,omitempty"`
+
+	// Additional containers of the component.
+	// +optional
+	AdditionalContainers []corev1.Container `json:"additionalContainers,omitempty"`
+
+	// Additional volumes of the component.
+	// +optional
+	AdditionalVolumes []corev1.Volume `json:"additionalVolumes,omitempty"`
```

Comment:
I have a question about how to consume additional volumes？

Reply:
That's a good question. @yeya24 WDYT?

Reply:
Good question! In my first thought, the `AdditionalVolumes` is only designed for the additional sidecar containers since we have the `AdditionalContainers` field so we can define the `volumeMount` there.

So it doesn't support to add additional `volumeMount` for the component itself. Maybe we can consider adding it in the future, but currently not needed.

Reply:
agreed, If you want to use sidecar to collect log or tracing, shared volume is required.

Reply:
"So it doesn't support to add additional volumeMount for the component itself (TiDB, TiKV, PD, etc).", make this clear in the comments?

Reply:
@DanielZhangQD Done.